src/Utils/ExecutableFinder.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Wkhtmltopdf\Utils;
+
+use Kdyby;
+use Nette;
+
+
+
+/**
+ * @author Filip Procházka <filip@prochazka.su>
+ */
+class ExecutableFinder extends Nette\Object
+{
+
+	/**
+	 * @var array
+	 */
+	public static $executables = array(
+		'wkhtmltopdf',
+		'wkhtmltopdf-amd64',
+		'wkhtmltopdf-i386'
+	);
+
+	/**
+	 * @var array
+	 */
+	public static $lookupPaths = array(
+		'/usr/local/sbin',
+		'/usr/local/bin',
+		'/usr/sbin',
+		'/usr/bin',
+		'/sbin',
+		'/bin',
+	);
+
+
+
+	/**
+	 * @throws \RuntimeException
+	 * @return string
+	 */
+	public function __toString()
```

Comment:
Tak jsem to měl původně ale nějak se mi to nelíbilo, nevymyslel jsem jak udělat aby se v configu dalo poskládat nějak pěkně do sebe